front-end/src/ui-components/Card.tsx

```diff
@@ -19,10 +19,8 @@ const Card = ({ className, children }:Props ) => {
 export default styled(Card)`
 	background-color: white;
 	padding: 2rem 3rem 2rem 3rem;
-	border-style: solid;
-	border-width: 1px;
-	border-color: grey_border;
 	border-radius: 3px;
+	box-shadow: box_shadow_card;
```

Comment:
Added a `box_shadow_card` style to theme. `box_shadow_secondary_grey` is used for buttons and I should probably rename it to `box_shadow_button_inset`.

Reply:
thanks